tests/urlpatterns_reverse/named_urls_conflict.py

```diff
@@ -0,0 +1,17 @@
+from django.conf.urls import url
+
+from .views import empty_view
+
+urlpatterns = [
+    # No kwargs
+    url(r'^conflict/$', empty_view, name="name-conflict"),
+
```

Comment:
I'd omit the blank lines since the comments are already breaking up the sections.